_data/KiranAdhikari/KiranAdhikari.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kiran Adhikari
+institution: Tribhuvan University
+profile_pic: kiranadhikari.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `kiranadhikari.jpg` to `kiranadhikari.png` in the markdown file.
2. Rename `kiranadhikari.png` to `kiranadhikari.jpg`.
I'd suggest you make the change in the markdown file!